kubernetes/configs/test/one_trust_notices.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+LANGS=it
+PUBLIC_STORAGE_PATH=one-trust-notices
+CONTENT_STORAGE_BUCKET=interop-privacy-notices-content-test
+HISTORY_STORAGE_BUCKET=interop-privacy-notices-history-test
+
+JOB_ONE_TRUST_NOTICES_RESOURCE_CPU="500m"
+JOB_ONE_TRUST_NOTICES_RESOURCE_MEM="1Gi"
```

Comment:
we should test, but probably it may run with
```suggestion
JOB_ONE_TRUST_NOTICES_RESOURCE_CPU="250m"
JOB_ONE_TRUST_NOTICES_RESOURCE_MEM="500Mi"
```

Reply:
I'll sincerely leave the default and test the memory requirements in an isolated test to be fair